docs/bigquery/snippets.py

```diff
@@ -1810,6 +1930,100 @@ def test_client_query_destination_table_cmek(client, to_delete):
     # [END bigquery_query_destination_table_cmek]
 
 
+def test_client_query_relax_column(client, to_delete):
+    dataset_id = 'query_relax_column_{}'.format(_millis())
+    dataset_ref = client.dataset(dataset_id)
+    dataset = bigquery.Dataset(dataset_ref)
+    dataset.location = 'US'
+    dataset = client.create_dataset(dataset)
+    to_delete.append(dataset)
+
+    # [START bigquery_relax_column_query_append]
+    # from google.cloud import bigquery
+    # client = bigquery.Client()
+    # dataset_ref = client.dataset('my_dataset')
+
+    table_ref = dataset_ref.table('my_table')
+    schema = [
+        bigquery.SchemaField('full_name', 'STRING', mode='REQUIRED'),
+        bigquery.SchemaField('age', 'INTEGER', mode='REQUIRED'),
+    ]
+    table = client.create_table(bigquery.Table(table_ref, schema=schema))
+
+    # Configure the query to append the results to a destination table,
+    # allowing field relaxation
+    job_config = bigquery.QueryJobConfig()
+    job_config.schema_update_options = [
+        bigquery.SchemaUpdateOption.ALLOW_FIELD_RELAXATION,
+    ]
+    job_config.destination = table_ref
+    job_config.write_disposition = bigquery.WriteDisposition.WRITE_APPEND
+
+    query_job = client.query(
+        # Write a query that will omit a required field
```

Comment:
Perhaps we demonstrate with all the columns projected but the value being NULL?  SELECT "Beyonce" as full_name, NULL as age?